source/whats-new.txt

```diff
@@ -54,6 +54,10 @@ The 5.0 driver release introduces the following features:
   name, or it defaults to ``MULTIPLE`` if you specify more than one
   host.
 
+- ``BsonDecimal128`` values now respond to method calls in the same way as Java
```

Comment:
S: remove "now" since this is already sorted into a versioned section
```suggestion
- ``BsonDecimal128`` values respond to method calls in the same way as Java
```